mars/services/task/execution/ray/executor.py

```diff
@@ -360,6 +380,19 @@ async def get_progress(self) -> float:
 
     async def cancel(self):
         """Cancel execution."""
+        subtask_num = len(self._output_object_refs_nums)
```

Comment:
The `execute_subtask_graph` should be cancelled first.

Reply:
I am not sure if all the backends of `execute_subtask_graph` already be cancelled in the task processor.

Reply:
We can check if the task is cancelled when submitting subtasks.